tests/test_uwsgi_plugin.py

```diff
@@ -0,0 +1,62 @@
+from testing.sandbox import install_fake_uwsgi
+install_fake_uwsgi()
```

Comment:
Very true. Hmm... taking a look at the yelp-wsgi-encoding I don't see any invocation (or reference) to uwsgi. It looks like they're building a wsgi app, but not actually running uwsgi itself right? We would actually have to run the tests from inside the uwsgi application to get the uwsgi_pymodule available I think :/